index.md

```diff
@@ -59,9 +59,11 @@ Non-public contact via email: [organizers email][contact]
 <div class="medium-4 columns">
 ### Upcoming
 
-* Wednesday, February 26th 2025
+[Calendar subscription link](https://emacs-berlin.org/calendar.ics)
+
+* Tuesday, March 11th 2025
```

Comment:
I would suggest to state the type of meetup next to each date here, to avoid confusion. So like this:

```suggestion
* Tuesday, March 11th 2025 (in-person only)
```

and similarly for all the following ones too.